index.html

```diff
@@ -1715,38 +1694,33 @@
             <em>connection</em>, it MUST run the following steps:
           </p>
           <ol>
-            <li>
-              <a>Queue a task</a> to run the following steps in order:
+            <li>If the <a>presentation connection state</a> of
+            <em>connection</em> is not <code>connected</code>, then abort these
+            steps.
+            </li>
+            <li>Otherwise, for each <em>known connection</em> in the <a>set of
+            presentations</a> in the <a>controlling user agent</a>:
               <ol>
-                <li>If the <a>presentation connection state</a> of
-                <em>connection</em> is not <code>connected</code>, then abort
-                these steps.
-                </li>
-                <li>Otherwise, for each <em>known connection</em> in the <a>set
-                of presentations</a> in the <a>controlling user agent</a>:
+                <li>If the <a>presentation identifier</a> of <em>known
+                connection</em> and <em>connection</em> are equal, and the <a>
+                  presentation connection state</a> of <em>known
+                  connection</em> is <code>connected</code>, then run the
+                  following steps:
                   <ol>
-                    <li>If the <a>presentation identifier</a> of <em>known
-                    connection</em> and <em>connection</em> are equal, and the
-                    <a>presentation connection state</a> of <em>known
-                    connection</em> is <code>connected</code>, then run the
-                    following steps:
-                      <ol>
-                        <li>Set <a>presentation connection state</a> of
-                        <em>known connection</em> to <code>terminated</code>.
-                        </li>
-                        <li>
-                          <a>Fire</a> an event named <code>statechange</code>
-                          at <em>known connection</em>.
-                        </li>
-                      </ol>
+                    <li>Set <a>presentation connection state</a> of <em>known
+                    connection</em> to <code>terminated</code>.
+                    </li>
+                    <li>
+                      <a>Queue a task</a> to <a>fire</a> an event named
```

Comment:
I fixed that. I note that the steps that tell the user agent to "fire" an event probably need to be updated as well to rather use "fire a simple event" (for events that use the Event interface) and more specific prose for the couple of specific events we have.

"Fire a simple event" also conveys the fact that the event is a trusted event which uses the Event interface, does not bubble and is not cancelable, whereas "Fire" does not say which event interface to use.